test/classes/claridge/baseConstructorCall.chpl

```diff
@@ -1,17 +1,18 @@
 class BaseClass {
   var a, b: int;
 
-  proc BaseClass(b: int) {
+  proc init(b: int) {
     this.b = b;
+    super.init();
```

Comment:
> I don't think we should force tests to use phase 1 for the purposes of finding bugs in the implementation if they don't "want" it naturally.

Put another way, if rewriting the tests exposes bugs, that's great, but the goal shouldn't be to rewrite the tests in a way that maximizes the chances that they expose bugs; it should be to rewrite them to work and be natural.  I'm guessing that even doing so, we'll find plenty of bugs and feature requests to keep us busy—particularly when we turn our attention to the module code.